Cargo.lock

```diff
@@ -0,0 +1,1614 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I think I will remove it as it's generated as a result of "workspace" Cargo.toml build we need just for GH actions to work 🤔